packages/components/modals/ModalBase.tsx

```diff
@@ -159,82 +161,88 @@ const ModalBase: React.FC<ModalBaseProps> = ({
+                justifyContent: "space-between",
+                width: "100%",
+                paddingHorizontal: modalMarginPadding,
+                paddingVertical: layout.spacing_x2,
+              }}
+            >
+              {(label || labelComponent || description) && (
+                <View
+                  style={{
+                    flex: 1,
+                    flexDirection: "row",
+                    alignItems: "center",
+                  }}
+                >
+                  {onBackPress && (
+                    <TouchableOpacity
+                      activeOpacity={0.9}
+                      style={{
+                        height: 32,
+                        width: 32,
+                        backgroundColor: neutral22,
+                        borderRadius: 20,
+                        alignItems: "center",
+                        justifyContent: "center",
+                        marginRight: 12,
+                      }}
+                      onPress={onBackPress}
+                    >
+                      <SVG source={chevronLeft} height={12} width={12} />
+                    </TouchableOpacity>
                   )}
 
-                  {labelComponent}
-
-                  {!!description && (
-                    <>
-                      <SpacerColumn size={1} />
-                      <BrandText
-                        style={[
-                          fontSemibold14,
-                          {
-                            color: neutral77,
-                            width: "100%",
-                            lineHeight: 20,
-                            flexWrap: "wrap",
-                          },
-                        ]}
-                      >
-                        {description}
+                  <View style={{ flex: 1, width: "100%" }}>
+                    {label && (
+                      <BrandText style={{ color: "white", lineHeight: 24 }}>
+                        {label}
                       </BrandText>
-                    </>
-                  )}
+                    )}
+
+                    {labelComponent}
+
+                    {description && (
+                      <>
+                        <SpacerColumn size={1} />
+                        <BrandText
+                          style={[
+                            fontSemibold14,
+                            {
+                              color: neutral77,
+                              width: "100%",
+                              lineHeight: 20,
+                              flexWrap: "wrap",
+                            },
+                          ]}
+                        >
+                          {description}
+                        </BrandText>
+                      </>
+                    )}
+                  </View>
                 </View>
-              </View>
-            )}
+              )}
 
-            {!!Header && <Header />}
+              {Header && <Header />}
```

Comment:
The `!!` ensures the resulting type is a boolean so you have to keep it 👍

Reply:
thks